tensorboard/webapp/metrics/views/right_pane/right_pane_test.ts

```diff
@@ -302,6 +311,53 @@ describe('metrics right_pane', () => {
     // mat-select does not render `input` or a DOM that can be manipulated.
     // skip the test for now.
 
+    describe('card width setting enabled', () => {
+      const CARD_WIDTH_SLIDER = '.card-width mat-slider';
+
+      beforeEach(() => {
+        store.overrideSelector(selectors.getEnabledCardWidthSetting, true);
+      });
+
+      it('renders slider and reset button', () => {
+        const fixture = TestBed.createComponent(SettingsViewContainer);
+        fixture.detectChanges();
+
+        const el = fixture.debugElement.query(By.css('.card-width'));
+        expect(el.query(By.css('mat-slider'))).toBeTruthy();
+        expect(el.query(By.css('button'))).toBeTruthy();
+
+        expect(getMatSliderValue(select(fixture, CARD_WIDTH_SLIDER))).toBe(
+          TEST_ONLY.MIN_CARD_WIDTH_SLIDER_VALUE.toString()
+        );
+      });
+
+      it('dispatches action when adjusting the slider', fakeAsync(() => {
+        const fixture = TestBed.createComponent(SettingsViewContainer);
+        fixture.detectChanges();
+        const slider = select(fixture, CARD_WIDTH_SLIDER);
+
+        // Adjust slider.
```

Comment:
`Adjusts` and I think the code should speak for itself so no need to comment here.

Reply:
Done.